src/lib/components/shell/Header.svelte

```diff
@@ -60,7 +60,8 @@
 
   <ShowHide let:show let:toggle>
     <Button form="text" class="print:hidden" onclick={toggle}>
-      <i class="far fa-question-circle" />
+      <i class="hidden sm:inline far fa-comment" />
```

Comment:
As seen in line 53, the breakpoint at which the FAQ button shows up is `lg` and not `sm` so that needs to determine the classes used here.